docker-storage-setup.sh

```diff
@@ -990,6 +989,10 @@ setup_docker_root_lv_fs() {
 }
 
 check_storage_options(){
+  if [ "$STORAGE_DRIVER" == "devicemapper" ] && [ -z "$CONTAINER_THINPOOL" ];then
+     Fatal "CONTAINER_THINPOOL must be defined for the devicemapper storage driver."
+  fi
+
```

Comment:
Right. So we have two backward compatibility cases.

- No options specified. This is docker case and it will not be broken
- Options specified and we will create docker-pool. Some fedora package might have pulled this code in. These are the users I am concerned about.